UPDATING.md

```diff
@@ -52,6 +52,10 @@ assists users migrating to a new version.
 
 ## Master
 
+### Marking success/failed automatically clears failed downstream tasks
+
+When marking a task success/failed in Graph View, its downstream tasks that are in failed/upstream_failed state are automatically cleared.
+
```

Comment:
@kaxil @ashb is this a 2.1 or 3.0 change? The behaviour is changing but it should not broke DAGs or pipelines (I think).

Reply:
I think 2.1